api/roleAccess/dtos/to-role-assignment-dto.ts

```diff
@@ -129,6 +129,7 @@ export function toDenySARoleAssignmentBody(
         requestDate: allocateRoleData.requestCreated,
         reviewer: currentUserId,
         infoRequired: allocateRoleData.accessReason === 'Request more information',
```

Comment:
Should we have infoRequired now David? If we presume, infoRequired true when the comment is present (not null) then we no longer need this field - field was only present because we previously were not concerned with the comment itself

Reply:
Comment can be empty so it is required